config/index.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     index: path.resolve(__dirname, '../dist/index.html'),
     assetsRoot: path.resolve(__dirname, '../dist'),
     assetsSubDirectory: 'static',
-    assetsPublicPath: '/',
+    assetsPublicPath: '/best-resume-ever',
```

Comment:
One more "/" needed here ` assetsPublicPath: '/best-resume-ever/'`, otherwise it will not work
Gh-pages will try to get assets from a path like this `/best-resume-everstatic` which would not be valid

Reply:
Oh my bad, I'm used to deploy on my personal domain... :/